### ISSUE TITLE
When events are declared on a Backbone View an error is being thrown if the event doesn't match the specified selector

### ISSUE THREAD

**Issue by trabianmatt** [trabian/titanium-backbone#10]
For example:

``` coffeescript
View = require 'tb/views/base'

$ = Backbone.$

module.exports = class Window extends View

  viewName: 'Window'

  attributes:
    layout: 'vertical'
    backgroundColor: '#'

  events:
    'click Button': 'showAlert'

  showAlert: ->
    alert 'click'

  render: ->

    $('<Button>')
      .attr('title', 'Testing')
      .appendTo @$el

    $('<Label>')
      .attr('text', 'Test Label')
      .appendTo @$el

    @
```

If I click on the button then the alert shows as expected. If I click anywhere else then I get the following error:

`Script Error = Can't find variable: context at app-impl.js (line 5017).`

The offending code is in closest():

``` javascript
      closest: function(selector) {
        var node;
        node = this[0];
        while (node && !_matches(node, selector)) {
          node = (node !== context) && node.parent;
        }
        return $(node);
      },
```

It looks like I accidentally copied over the 'context' variable even though we're not using context yet.

**Comment by trabianmatt** [comment]
Instead of just removing it I may take a stab at implementing support for context.

**Comment by trabianmatt** [comment]
This was fixed in b05d5b24d by removing the check for context. In jQuery and Zepto, the default context is 'document' so an optional context scope makes sense. At the moment, however, there is no default context in titanium-backbone (all traversal methods start at $(el)) so we don't need to keep track of the context.
